pkgs/applications/networking/sync/rclone/default.nix

```diff
@@ -17,9 +20,13 @@ buildGoModule rec {
 
   outputs = [ "out" "man" ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  buildInputs = lib.optional enableCmount (if stdenv.isDarwin then macfuse-stubs else fuse);
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
 
-  buildFlagsArray = [ "-ldflags=-s -w -X github.com/rclone/rclone/fs.Version=${version}" ];
+  buildFlagsArray = [
+    "-tags=cmount"
```

Comment:
This will not be disabled if I set enableCmount to false.

Reply:
Missed that, thanks.